app/scenes/Document/components/Document.tsx

```diff
@@ -617,6 +616,8 @@ type ContentsContainerProps = {
 };
 
 const ContentsContainer = styled.div<ContentsContainerProps>`
+  margin-top: calc(44px + 6vh);
```

Comment:
@tommoor Curious about this coz it breaks the TOC alignment with the title 🤔 

**Normal Page**
![Screenshot 2024-06-16 at 10 18 13 PM](https://github.com/outline/outline/assets/132386067/fa8b0992-8c6d-4109-86e3-ed7ec61a0fd1)

**Deleted Page**
![Screenshot 2024-06-16 at 10 18 26 PM](https://github.com/outline/outline/assets/132386067/572b9093-5b32-434d-89db-4ce45794b6bc)

Reply:
I want it always aligned with the metadata beneath the title, sorry should have been clearer

Reply:
Ahhh cool! :)